src/containers/Config.js

```diff
@@ -2,7 +2,8 @@ export const Config = {
   centerPoint: [37.73, 14.2],
   initialZoom: 8,
   maxZoom: 18,
-  tileURL: 'http://tile.stamen.com/terrain/{z}/{x}/{y}.jpg',
+  tileURL:
+    'https://api.mapbox.com/styles/v1/marshallr2/cjmgpleyx0lri2tph0orws1m5/tiles/256/{z}/{x}/{y}@2x?access_token=',
```

Comment:
if this URL is specific to you account then let's create a generic account for the project.

Reply:
I just confirmed - using this URL and my access token I get 404.

Reply:
I made it public. If you could test to ensure that fixes the issue then we can move the map to an account name fitting the project.

Reply:
It works! Let's create the new account and update the URL